### PR TITLE
feat(land-covers): add land cover documentation

### PR DESCRIPTION
**Pull request by daniel-buchanan**
----

**Notes:**
This PR adds Land Covers to the public facing documentation.

----

## PR Checklist
- [ ] Documentation has been checked by at least one team member
- [x] Links work to navigate documentation
- [x] Images stored in `_images`
- [x] Spell checked

> If some of the above are not completed why?
> **REASON HERE**

### CODE REVIEW

File: docs/resource-types/OGC/Collections.md [hunk 1]
```diff
@@ -1,79 +0,0 @@
...
-taxonomy:
-    category:
-        - api
-        - data
-    post_tag:
-        - api
-
----
-
-# Resource Type: OGC – Collections
```
Comment (danielR9) [inline docs/resource-types/OGC/Collections.md:16]: Why are the OGC docs being removed?

File: docs/resource-types/core/land-covers.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  "classificationDescription": "string",
+  "eligibility": {...},
+  "landCoverClassification": {...},
+  "activities": [...],
+  "classifications": [...],
+  "centroid": {...},
+  "spatialFeature": {...},
+  "totalArea": {...},
+  "totalLength": {...},
+  "internalId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
```
Comment (danielR9) [inline docs/resource-types/core/land-covers.md:68]: Can we not put internalId in here? Its not in the table below, and it shouldn't really be in the response either (but that can't be fixed from here)

File: docs/resource-types/core/land-covers.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  "id": "string",
+  "meta": {...},
+  "self": "string",
+  "identifiers": [...],
+  "links": [...],
+  "name": "string",
+  "resourceType": "string"
```
Comment (danielR9) [inline docs/resource-types/core/land-covers.md:75]:
```suggestion
  "resourceType": "/core/land-cover"
```

File: docs/resource-types/core/land-covers.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+{
+  "id": {
+    "id": "string",
+    "scheme": "string"
+  },
+  "isPrimary": true,
+  "name": "string",
+  "productiveArea": {
+    "area": 0,
+    "units": 0
```
Comment (danielR9) [inline docs/resource-types/core/land-covers.md:142]:
```suggestion
    "units": "string"
```

Comment (danielR9) [reply docs/resource-types/core/land-covers.md:142]: And if we are defining the total-area elsewhere, then we can just do `{...}` here?

File: docs/resource-types/OGC/Collections.md (hunk 1 above)
Comment (daniel-buchanan) [reply docs/resource-types/OGC/Collections.md:16]: @danielR9 they got moved to `/docs/ogc-api`

File: docs/resource-types/core/land-covers.md (hunk 2 above)
Comment (daniel-buchanan) [reply docs/resource-types/core/land-covers.md]: @danielR9  removed.

**Review by danielR9 — APPROVED** [review]

**Pull request merged**